components/homme/src/repro_sum_mod.F90

```diff
@@ -41,7 +41,7 @@ module repro_sum_mod
 #else
    use shr_kind_mod,  only: r8 => shr_kind_r8, i8 => shr_kind_i8 
 #endif
-   use shr_sys_mod,   only: shr_sys_abort  
+   use shr_abort_mod,   only: shr_sys_abort=>shr_abort_abort
```

Comment:
This is the single difference between the modules that were in the csm_share folder in homme, and the modules in `$repo/share/utils`.